### PR TITLE
Apps 1551 adding text processing

### PR DESCRIPTION
**Pull request by RoniDataloopAi**

### CODE REVIEW

File: modules/pdf/pdf_extract/pdf_extractor.py [hunk 1]
```diff
@@ -132,3 +127,10 @@ def extract_images_from_pdf(pdf_path) -> List:
                     logger.info(f"Image saved as {image_name}")
 
         return images_paths
+
+
+if __name__ == '__main__':
+    dl.setenv('rc')
```
Comment (ShadiMah) [inline modules/pdf/pdf_extract/pdf_extractor.py:133]: clean main

File: modules/txt/chunking/chunks_extractor.py [hunk 2]
```diff
@@ -218,14 +205,7 @@ def clean_multiple_chunks(self, items: [dl.Item], context: dl.Context) -> List[d
         remote_path_for_clean_chunks = node.metadata['customNodeConfig']['remote_path_for_clean_chunks']
         # local test
```
Comment (ShadiMah) [inline modules/txt/chunking/chunks_extractor.py:206]: delete all local test comments (All modules)

**Review by ShadiMah — CHANGES_REQUESTED** [review]

**Review by orshabtay — APPROVED** [review]

File: modules/txt/chunking/chunks_extractor.py (hunk 2 above)
Comment (RoniDataloopAi) [reply modules/txt/chunking/chunks_extractor.py]: deleted

File: modules/pdf/pdf_extract/pdf_extractor.py (hunk 1 above)
Comment (RoniDataloopAi) [reply modules/pdf/pdf_extract/pdf_extractor.py]: deleted

**Review by ShadiMah — APPROVED** [review]

**Pull request merged**